src/ui/App.tsx

```
@@ -8,6 +8,7 @@ import { Notifications } from "@mantine/notifications";
 
 import ColorSchemeContext from "./ColorSchemeContext";
 import { Router } from "./Router";
+import { UserResolverProvider } from "./components/NameOptionalCard";
```

ESLint

[error] Missing file extension for "./components/NameOptionalCard"

(import/extensions)